test/checkpoint/test_checkpoint.c

```diff
@@ -193,6 +229,10 @@ main(int argc, char *argv[])
     g.ts_oldest = 1;
 
     printf("%s: process %" PRIu64 "\n", progname, (uint64_t)getpid());
+    if (g.predictable_replay)
```

Comment:
Print all the arguments that could change the results.

Reply:
I thought of improving this more generically across all predictable tests: WT-10938